src/models/MyAccount.ts

```diff
@@ -96,7 +96,7 @@ export default class MyAccount {
           (subfield: { tag: string; subfield: string }) => subfield.tag === "a"
         ).content
         isResearch = nineTenContent.startsWith("RL")
-        isNyplOwned = nineTenContent !== "RLOTF"
+        isNyplOwned = !isResearch || nineTenContent !== "RLOTF"
```

Comment:
What does RLOTF stand for?...

Reply:
"Research Library On The Fly". It's a code we add to OTF (aka "virtual") records, to tag them as being Research OTF records.

Reply:
I was thinking this line could use a comment to explain to future devs

Reply:
Nitpick on the comment - RL is the 910 content for all research materials, OTF is added specifically for virtual records.